gradle.properties

```diff
@@ -1,8 +1,8 @@
 # Kotlin
 version=1.9.0-SNAPSHOT
 group=org.jetbrains.kotlinx
-kotlin_version=2.0.0
-kotlin_language_version=2.0
+kotlin_version=2.1.0
+kotlin_language_version=2.1
```

Comment:
Does this line do anything useful today other than accidentally disabling the "warnings are errors" flag?

Reply:
AFAIR release team uses it, but not sure. Will figure out separately

Reply:
The release team may override this property, this I get, but do we need to keep it in gradle.properties?

Reply:
At least for the sake of having "release team" branch and mainline having the same configuration and catching the same warnings/errors.